server/protocols/jmap-rfc-8621/doc/specs/spec/mail/message.mdown

```diff
@@ -852,24 +852,25 @@ Due to the format of the Email object, when creating an Email there are a number
 - Header fields beginning with `Content-` MUST NOT be specified on the Email
   object, only on EmailBodyPart objects.
 - If a bodyStructure property is given, there MUST NOT be textBody, htmlBody, or
-  attachments properties.
+  attachments properties. <aside class="warning">Not implemented. *bodyStructure* property is not supported in Email/set create.</aside>
 - If given, the bodyStructure EmailBodyPart MUST NOT contain a property
   representing a header field that is already defined on the top-level Email
-  object.
+  object. <aside class="warning">Not implemented. *bodyStructure* property is not supported in Email/set create.</aside>
 - If given, textBody MUST contain exactly one body part and it MUST be of type
-  `text/plain`.
+  `text/plain`. <aside class="warning">Not implemented. *textBody* property is not supported in Email/set create.</aside>
 - If given, htmlBody MUST contain exactly one body part and it MUST be of type
   `text/html`.
 - Within an EmailBodyPart:
-  - The client may specify a partId OR a blobId, but not both. If a partId is
-    given, this partId MUST be present in the bodyValues property.
+  - The client may specify a *partId* OR a *blobId*, but not both. If a *partId* is
+    given, this *partId* MUST be present in the bodyValues property. <aside>htmlBody part needs to have a *partId*. attachments parts need to specify *blobId*. </aside>
```

Comment:
s/attachments/Attachments

Reply:
That's intendeed to match the property case.

Maybe we can wrap property names in code quotes like `attachments` ?